guides/source/autoloading_and_reloading_constants.md

```diff
@@ -389,7 +389,7 @@ Customizing Inflections
 
 By default, Rails uses `String#camelize` to know which constant a given file or directory name should define. For example, `posts_controller.rb` should define `PostsController` because that is what `"posts_controller".camelize` returns.
 
-It could be the case that some particular file or directory name does not get inflected as you want. For instance, `html_parser.rb` is expected to define `HtmlParser` by default. What if you prefer the class to be `HTMLParser`? There are a few ways to customize this.
+It could be the case that some particular file or directory name does not get inflected as wanted. For instance, `html_parser.rb` is expected to define `HtmlParser` by default. What if `HTMLParser` is preferred? There are a few ways to customize this.
```

Comment:
```suggestion
It could be the case that some particular file or directory name does not get inflected as desired. For instance, `html_parser.rb` is expected to define `HtmlParser` by default. What if `HTMLParser` is preferred? There are a few ways to customize this.
```